skills/playwright-regression-testing/references/regression-catalogs.md

```
@@ -0,0 +1,31 @@
+# Regression Catalogs
+
+## Tier Model
+
+```
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)